### PR TITLE
Update app_en.arb

### PR DESCRIPTION
**Pull request by korablique**

### CODE REVIEW

File: lib/l10n/app_en.arb [hunk 1]
```diff
@@ -192,7 +192,7 @@
     "@map_page_loading_shops_hint1": {
         "description": "Hint shown nearby the progress bar on the map page. The hint explains what is being loaded."
     },
-    "map_page_loading_shops_hint2": "Still loading stores - seems the area has a big number of organizations",
+    "map_page_loading_shops_hint2": "Still loading stores - seems the area has a big number of them",
```
Comment (cyclomaticcomplexity1981) [inline lib/l10n/app_en.arb:195]: not sure about this, i think i would be better if we just used "stores"

"Still loading stores - seems there are a lot of stores in this area"

File: lib/l10n/app_en.arb [hunk 2]
```diff
@@ -254,9 +254,9 @@
     "shop_product_range_page_add_product": "Add a product",
     "shop_product_range_page_have_you_seen_product_here": "Do they sell it here?",
     "shop_product_range_page_product_last_seen_here": "Last seen here: ",
-    "shop_product_range_page_confirmed_products_country": "Products seen in this store by other users of Plante",
+    "shop_product_range_page_confirmed_products_country": "Products seen in this store by users of Plante",
```
Comment (cyclomaticcomplexity1981) [inline lib/l10n/app_en.arb:257]: i like this one

File: lib/l10n/app_en.arb [hunk 3]
```diff
@@ -223,7 +223,7 @@
     "mod_reason_non_vegan_practices_but_helps_veganism": "Product's manufacturing process utilizes non-vegan practices. But the product is considered ambiguous by Plante because all ingredients are vegan, and its mainstream appeal to help the vegan movement grow.",
     "mod_reason_possibly_cane_sugar_in_ingredients": "Product contains sugar, which could be obtained from sugarcane and processed through Bone Black/Bone Char.",
     "mod_reason_some_ingredient_has_unknown_origin": "Some of the ingredients have unknown origin",
-    "mod_reason_some_ingredient_is_in_fact_a_category2": "Ingredients labeled as \"Flavors\", \"Natural Dyes\", \"Additives\" (or in a similar ambiguous fashion) come from either plant, animal, or mineral sources. As such, products with such ingredients are (sometimes intentionally) ambiguous in their suitability for vegans.",
+    "mod_reason_some_ingredient_is_in_fact_a_category2": "Ingredients labeled as \"Flavors\", \"Natural Dyes\", \"Additives\" (or in a similar ambiguous fashion) come from either plant, animal, or mineral sources.",
```
Comment (cyclomaticcomplexity1981) [inline lib/l10n/app_en.arb:226]: I can follow this one, i guess we'll have to see if it's still clear for the users.

File: lib/l10n/app_en.arb [hunk 4]
```diff
@@ -192,11 +192,11 @@
     "@map_page_loading_shops_hint1": {
         "description": "Hint shown nearby the progress bar on the map page. The hint explains what is being loaded."
     },
-    "map_page_loading_shops_hint2": "Still loading stores - seems the area has a big number of organizations",
+    "map_page_loading_shops_hint2": "Still loading stores - seems there are a lot of stores in this area",
     "@map_page_loading_shops_hint2": {
         "description": "Hint shown nearby the progress bar on the map page. The hint explains what is being loaded."
     },
-    "map_page_loading_shops_hint3": "Still loading stores - it takes longer than usual, sorry!",
+    "map_page_loading_shops_hint3": "Loading stores seems to take longer then expected, thank you for being patient",
```
Comment (blazern) [inline lib/l10n/app_en.arb:199]:
```suggestion
    "map_page_loading_shops_hint3": "Loading stores takes longer than expected, thank you for being patient",
```

**Review by blazern — CHANGES_REQUESTED** [review]

**Review by blazern — APPROVED** [review]

**Review by cyclomaticcomplexity1981 — APPROVED** [review]

**Review by cyclomaticcomplexity1981 — APPROVED** [review]
👍

**Review by cyclomaticcomplexity1981 — APPROVED** [review]

**Pull request merged**